x-pack/plugins/observability/public/components/app/observability_status/observability_status_box.tsx

```diff
@@ -0,0 +1,123 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import {
+  EuiBadge,
+  EuiButton,
+  EuiButtonEmpty,
+  EuiFlexGroup,
+  EuiFlexItem,
+  EuiIcon,
+  EuiPanel,
+  EuiText,
+  EuiTitle,
+} from '@elastic/eui';
+import React from 'react';
+import { FormattedMessage } from '@kbn/i18n-react';
+
+interface Props {
+  dataSource: string;
```

Comment:
My instinctive expectation of `dataSource` is that it will contain some sort of index pattern, data view, etc, but I think we use it as the title of the box.

If that's the case I think `title` is clearer, but maybe I'm missing something

Reply:
I didn't think much about namings because it will probably change when content is more defined, but you are right is misleading. I change it to `dataSourceName` for now, so is not tied to where is used.